config/core/deployments/webhook.yaml

```diff
@@ -125,8 +134,10 @@ apiVersion: v1
 kind: Service
 metadata:
   labels:
-    role: webhook
```

Comment:
leave old `role: webhook` label - otherwise the selector doesn't work

Reply:
Shouldn't matter. 

spec.selector has that label pair

Reply:
Yeah I was wrong and it doesn't affect the selector but can we leave it back in place and remove its say in a subsequent release after we give notice

Reply:
fixed